Source/Scene/PolylineCollection.js

```diff
@@ -1049,27 +1042,31 @@ import SceneMode from './SceneMode.js';
     function removePolylines(collection) {
         if (collection._polylinesRemoved) {
             collection._polylinesRemoved = false;
-
-            var polylines = [];
+            var definedPolylines = [];
+            var polyIndex = 0;
+            var polyline;
 
             var length = collection._polylines.length;
-            for ( var i = 0, j = 0; i < length; ++i) {
-                var polyline = collection._polylines[i];
-                if (defined(polyline)) {
-                    polyline._index = j++;
-                    polylines.push(polyline);
+            var definedPolylinesToUpdate = [];
```

Comment:
Small nitpick, but you can you move `var definedPolylinesToUpdate = [];` so it's placed after `var definedPolylines = [];`?